CONTRIBUTING.md

```diff
@@ -42,8 +42,8 @@ To run commands like `gem install` from the repo:
 To run bundler test:
 
     $ cd bundler
-    $ bin/rake spec:deps
-    $ bin/rspec spec
+    $ RUBYOPT='-I../lib' bin/rake spec:deps
+    $ RUBYOPT='-I../lib' bin/rspec spec
```

Comment:
Seems like by default the repo bundler runs against Ruby's rubygems instead of the repo one, so I had to do this to pick my changes.

Just a note for now but feels worthy of adding to the doc.

Reply:
Yes, thanks @lloeki, this should be documented, definitely. What I usually do is `export RGV=..`, which is what tests do, and make sure all development binstubs automatically run pointing to the correct copy.